### PR TITLE
Update examples to match new API

### LINKED ISSUE #20: Update examples
We need to update the included examples to use the new API.

### PR DESCRIPTION
**Pull request by ryanbmilbourne**
In addition to fixing the javascript examples, this commit also adds
example files showing their equivalent TCL Expect scripts.

Note that stripColors was removed.  Strip colors is not implemented in
TCL Expect, and a future commit will remove it from from the list of
valid options.

Resolves #20

Increment PATCH version

### CODE REVIEW

File: examples/echo.tcl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env tclsh
+
+# echo.tcl: Simple example for using `echo` command with Expect.
+#
+# (C) 2015, Greg Cochard, Ryan Milbourne, ViaSat Inc.
```
Comment (gcochard) [inline examples/echo.tcl]: Remove my name from this one. I did not contribute to it :smile:

**Comment by gcochard** [comment]
Remove me from all files which are solely yours, and I'll merge this.

**Comment by ryanbmilbourne** [comment]
Ha, you're right.  Copied the Copyright line from the other examples.  Whoops, will fix.

**Comment by ryanbmilbourne** [comment]
@gcochard removed your name from Copyright headers in `examples/*.tcl`.

**Comment by gcochard** [comment]
:+1: :package: :ship: :roller_coaster: :rocket: :shipit:

**Pull request merged**